common/persistence/persistence-tests/cluster_metadata_manager.go

```diff
@@ -36,7 +36,7 @@ func (s *ClusterMetadataManagerSuite) SetupSuite() {
 func (s *ClusterMetadataManagerSuite) SetupTest() {
 	// Have to define our overridden assertions in the test setup. If we did it earlier, s.T() will return nil
 	s.Assertions = require.New(s.T())
-	s.ctx, s.cancel = context.WithTimeout(context.Background(), 30*time.Second*debug.TimeoutMultiplier)
+	s.ctx, s.cancel = context.WithTimeout(context.Background(), 30*time.Second*debugtimeout.Multiplier)
```

Comment:
I think this one and those below are antipatterns.  This IS a test by definition, and doesn't need a multiplier.  It should set the timeout it needs.  If 300 seconds is the right answer here it should be coded that way, it would be clearer.